client/src/js/components/DashBoard/RecipeDetails/Content/Reviews.jsx

```diff
@@ -102,31 +102,23 @@ class Reviews extends React.Component {
               <span className="float-right review-time">
                 {moment(new Date(review.createdAt)).fromNow()}
               </span>
-              <span className="float-right delete-review">
-                <i
-                  className="fa fa-trash fa-1x text-danger"
-                  role="button"
-                  tabIndex="-1"
-                  onKeyPress={this.onKeyPress}
-                  onClick={() => this.onDelete(review.id)}
-                />
-              </span>
+              { review.userId === parseInt(localStorage.getItem('userId'), 10) ?
+                <span className="float-right delete-review">
+                  <i
+                    className="fa fa-trash fa-1x text-danger"
+                    role="button"
+                    tabIndex="-1"
+                    onKeyPress={this.onKeyPress}
+                    onClick={() => this.onDelete(review.id)}
+                  />
+                </span>
+              : ''}
               <div style={{ clear: 'both' }} />
             </div>
-            <p style={{ marginTop: '9px', fontSize: '24px', fontWeight: 200 }}>{review.reviews}</p>
+            <p style={{ marginTop: '9px', fontSize: '24px', fontWeight: 200 }}>
```

Comment:
### Nitpick

Be weary of inline styling in your react. It can be a problem as your css grows bigger.
Infact I'd advise you shouldn't unless absolutely necessary.